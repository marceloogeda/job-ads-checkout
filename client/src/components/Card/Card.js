// @flow
import React, { type Node } from 'react';
import styled from 'styled-components';

const Card = styled.div`
  background-color: #fff;
  margin-bottom: 20px;
  padding: 20px;
  position: relative;
`;

export type CardProps = {
  children: Node
};

export default ({ children }: CardProps) => <Card>{children}</Card>;
