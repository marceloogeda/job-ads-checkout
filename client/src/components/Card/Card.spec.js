import React from 'react';
import TestRenderer from 'react-test-renderer';

import Card from './Card';

describe('# Card', () => {
  it('should render correctly', () => {
    const testRenderer = TestRenderer.create(<Card />);

    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
