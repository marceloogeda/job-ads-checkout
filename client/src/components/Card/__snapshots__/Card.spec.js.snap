// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# Card should render correctly 1`] = `
<div
  className="sc-bdVaJa cOxDyi"
/>
`;
